### PR TITLE
Create buddhism-and-the-environment_edelglass.md

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/articles/buddhism-and-the-environment_edelglass.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+course: nature
+tags:
+  - landscape
+  - mountains
+  - ritual
+year: 2021
+month: jun
+journal: orer
+publisher: oxford
+address: "Oxford"
```
Comment (buddhist-uni) [inline _content/articles/buddhism-and-the-environment_edelglass.md:19]: needs a page count also:

```
pages: 21
```

File: _content/articles/buddhism-and-the-environment_edelglass.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+---
+> Instead of focusing primarily on universal concepts found in ancient texts, scholars are just as likely to look at how local communities have drawn on Buddhist ontology, ethics, cosmology, symbolism, and rituals to develop Buddhist responses to local environmental needs, developing contemporary Buddhist environmentalisms
+In this article, the author reviews the current state of "eco-Buddhism", prviding the arguments for and against the idea.
```
Comment (buddhist-uni) [inline _content/articles/buddhism-and-the-environment_edelglass.md:22]: "providing" was mistyped and Markdown needs a blank line between paragraphs.

Also, the article provides a good summary of various traditional aspects of Buddhism's relationship with the environment as well which is worth calling out in the blurb 😊

File: _content/articles/buddhism-and-the-environment_edelglass.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+title: "Buddhism and the Environment"
+authors:
+  - "William Edelglass"
+external_url: "https://www.researchgate.net/profile/William-Edelglass/publication/354058724_Buddhism_and_the_Environment/links/6121c6f90c2bfa282a60482d/Buddhism-and-the-Environment.pdf"
+source_url: "https://doi.org/10.1093/acrefore/9780199340378.013.721"
+drive_links:
+  - "https://drive.google.com/file/d/17afaF0hfOtjnfNaphVt1dof-aooNBFyn/view?usp=share_link"
+formats: [pdf]
+course: nature
+tags:
```
Comment (buddhist-uni) [inline _content/articles/buddhism-and-the-environment_edelglass.md:11]: Let's maybe add `roots` (the history tag) as this is also a nice overview of Buddhist history in a way.

**Review by buddhist-uni — COMMENTED** [review]
Thanks for the add! Must be getting close to finishing the folder, eh?

**Comment by JJM317** [comment]
Ya, must be getting close. I've started working from the bottom up to find
materials faster.

~Joe
----------------

**Pull request merged**